batch.go

```diff
@@ -44,7 +45,7 @@ var batchCommand = &cli.Command{
 		&cli.StringFlag{
 			Name:        "in",
 			Required:    true,
-			Usage:       "Path of directory containing plot definitions.",
+			Usage:       "Path of directory containing processing profile definitions.",
```

Comment:
We could put the profiles into a single profiles.yaml file in the config folder (already specified with `--conf`). Remove the `--in` option completely and let it be handled with the config.

Reply:
That's a good idea 👍